src/main/groovy/edu/oregonstate/mist/catalogapitest/resources/InstructorResource.groovy

```diff
@@ -0,0 +1,210 @@
+package edu.oregonstate.mist.catalogapitest.resources
+
+import edu.oregonstate.mist.catalogapitest.core.ErrorPOJO
+import edu.oregonstate.mist.catalogapitest.core.Instructor
+import edu.oregonstate.mist.catalogapitest.db.InstructorDAO
+import io.dropwizard.jersey.params.IntParam
+import com.google.common.base.Optional
+import javax.validation.Valid
+import javax.ws.rs.Consumes
+import javax.ws.rs.DELETE
+import javax.ws.rs.GET
+import javax.ws.rs.POST
+import javax.ws.rs.PUT
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.WebApplicationException
+import javax.ws.rs.core.Context
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+import javax.ws.rs.core.UriInfo
+import org.skife.jdbi.v2.exceptions.UnableToExecuteStatementException
+
+/**
+ * Instructor resource class.
+ */
+@Path("/instructor")
+@Produces(MediaType.APPLICATION_JSON)
+class InstructorResource {
+
+    // Instantiate DAO -------------------------------------------------------------------------------------------------
+    private final InstructorDAO instructorDAO
```

Comment:
the comment is misleading because this statement allocates a variable but does not construct an object